common/src/test/scala/uk/ac/wellcome/platform/archive/common/generators/StorageManifestGenerators.scala

```diff
@@ -38,21 +37,21 @@ trait StorageManifestGenerators
       space = space,
       info = bagInfo,
       version = version,
-      manifest = BagManifest(
+      manifest = FileManifest(
         checksumAlgorithm,
         files = Seq(
           createBagFile,
           createBagFile,
           createBagFile
-        )
+        ).map { StorageManifestFile.apply }
       ),
-      tagManifest = BagManifest(
+      tagManifest = FileManifest(
         checksumAlgorithm,
         files = Seq(
           createBagFile,
           createBagFile,
           createBagFile
-        )
+        ).map { StorageManifestFile.apply }
```

Comment:
As a style thing I would not make this in-line as I reckon it's easier to read.

Reply:
oh hold on, that shouldn't be there

Reply:
This was temporary code to keep that generator compiling while I was fiddling elsewhere, but I meant to take it out